src/@sdk/index.ts

```diff
@@ -62,122 +67,104 @@ export const createSaleorClient = (url?: string, cache = new InMemoryCache()) =>
   });
 
 export class SaleorAPI {
-  getAttributes = this.watchQuery(QUERIES.Attributes, data => data.attributes);
+  getAttributes = this.watchQuery(
```

Comment:
Can we just remove whats not longer needded without introducing lots of code style changes?

Reply:
No idea why this change occured... I'll investigate it

Reply:
Updated vscode plugin and format on save probably

Reply:
yes, exactly 👍